tests/metaworld/envs/test_multitask_env.py

```diff
@@ -222,4 +221,44 @@ def test_task_name():
     assert sorted(env.all_task_names) == sorted(task_names)
 
     _, _, _, info = env.step(env.action_space.sample())
-    assert info['task_name'] in task_names
\ No newline at end of file
+    assert info['task_name'] in task_names
+
+@pytest.mark.parametrize('observation_type', ['plain', 'with_goal_id', 'with_goal_and_id', 'with_goal'])
+def test_observation_space(observation_type):
+    env_cls_dict = {'pick-place-v1': MEDIUM_MODE_CLS_DICT['train']['pick-place-v1']}
+    env_args_kwargs = {key: MEDIUM_MODE_ARGS_KWARGS['train'][key]
+                       for key in env_cls_dict.keys()}
+    multi_task_env = MultiClassMultiTaskEnv(
+        task_env_cls_dict=env_cls_dict,
+        task_args_kwargs=env_args_kwargs,
+        sample_goals=True,
+        obs_type=observation_type,
+    )
+    if observation_type == 'plain':
+        assert multi_task_env.observation_space.shape == (9, )
+    elif observation_type == 'with_goal_id':
+        assert multi_task_env.observation_space.shape == (59, )
+    elif observation_type == 'with_goal_and_id':
+        assert multi_task_env.observation_space.shape == (62, )
+    elif observation_type == 'with_goal':
+        assert multi_task_env.observation_space.shape == (12, )
+
+
+def test_action_space():
+    env_cls_dict = {'pick-place-v1': MEDIUM_MODE_CLS_DICT['train']['pick-place-v1']}
+    env_args_kwargs = {key: MEDIUM_MODE_ARGS_KWARGS['train'][key]
+                       for key in env_cls_dict.keys()}
+    multi_task_env = MultiClassMultiTaskEnv(
+        task_env_cls_dict=env_cls_dict,
+        task_args_kwargs=env_args_kwargs,
+        sample_goals=True,
+        obs_type="plain",
+    )
+    assert multi_task_env.action_space.shape == (4, )
+
+
+@pytest.mark.parametrize('task_name', list(_static_task_ids.keys())[:10])
```

Comment:
why does it skip the first 10?

Reply:
Doesn't this notation mean that the test will only be run on the first 10 environments in _static_task_ids?

Reply:
also the reasoning was that I didn't want to run the test on every metaworld environment (to save time)